docs/run-platform-server/installing.mdx

```diff
@@ -9,10 +9,10 @@ To install Horizon in production or non-development environments, we recommend t
 
 - bare-metal:
   - if host is debian linux, install prebuilt binaries [from repositories](#package-manager) using package manager.
-  - for any other hosts, download [prebuilt release binaries](#prebuilt-releases) of Stellar Horizon and Core for host target architecture and operation system.
+  - for any other hosts, download [prebuilt release binaries](#prebuilt-releases) of Stellar Horizon and Core for host target architecture and operation system or [compile from the source](https://github.com/stellar/go/blob/master/services/horizon/internal/docs/GUIDE_FOR_DEVELOPERS.md#building-horizon).
 - containerized:
-  - non-Helm Chart, if the target envrironment for container to run does not support Helm chart usage, run the prebuilt docker image of Horizon published on [dockerhub.com/stellar/horizon](https://hub.docker.com/r/stellar/stellar-horizon).
-  - Helm charts, when the target envrionment uses container orchestration such as Kubernetes and has enabled Helm Charts on cluster. The [Horizon Helm chart](https://github.com/stellar/helm-charts/tree/main/charts/horizon) manages installation life cycle. Use the [Helm Install command](https://helm.sh/docs/helm/helm_install/), it will accept Horizon's configuration parameters. Please review [Configuration](./configuring.mdx) first, to identify any specific configuration params needed.
+  - non-Helm Chart, if the target environment for container does not support Helm chart installation, pull the prebuilt image [dockerhub.com/stellar/stellar-horizon](https://hub.docker.com/r/stellar/stellar-horizon) with docker daemon on host. Choose the tag of that image for specific release version and then pull with `docker pull stellar/stellar-horizon#<tag>` onto host.
+  - Helm charts, when the target envrionment uses container orchestration such as Kubernetes. The [Horizon Helm chart](https://github.com/stellar/helm-charts/tree/main/charts/horizon) manages installation life cycle.
```

Comment:
```suggestion
  - **Helm Charts**: when the target envrionment uses container orchestration such as Kubernetes. The [Horizon Helm chart](https://github.com/stellar/helm-charts/tree/main/charts/horizon) manages installation life cycle.
```

Reply:
I  re-worked this with Orchestrated and Non-Orchestrated usage instead, let me know if that reads better.